project/README.md.twig

```diff
@@ -15,6 +15,14 @@ Branch | Travis | Coveralls | Scrutinizer |
 {{ stable_branch }}   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     | [![Scrutinizer Status][scrutinizer_stable_badge]][scrutinizer_stable_link] |
 {{ unstable_branch }} | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] | [![Scrutinizer Status][scrutinizer_unstable_badge]][scrutinizer_unstable_link] |
 
+{% if is_abandoned %}
+## WARNING: This bundle is abandoned
```

Comment:
```suggestion
## WARNING: This project is abandoned
```

Reply:
What about "package"? Some small packages maybe aren't a project itself (bridges, bundle implementations of a library, etc).

Reply:
This thing 😄

Reply:
Yeah let's go with repository, because we do have projects that are not packages (the sandbox)

Reply:
From the [Composer perspective](https://getcomposer.org/doc/04-schema.md#type), all the repositories are considered packages (then, there are package types, like libraries and so).

Reply:
Ok I'm fine with both

Reply:
I feel like more logic the fact that
- The repository will say that the repository is abandoned
- The package will say that the package is abandoned

This will also say that we will not necessarily be as quick to respond as on another repository.